site/docs/foundations/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Foundations
+layout: DetailTechnical
+sidebar:
+  label: Overview
+---
+
+Foundations are the fundamental principles that guide the creation of a cohesive design language, enabling you to build products that are easy to use and visually appealing.
+
+<GridLayout columns={{ xs: 1, sm: 2 }}>
```

Comment:
Should there be a Card for each foundation (in the sidebar)?  I believe there is in designs, but the design has less in the sidebar. unless I am looking at the wrong mock up

Reply:
also, is the beta tag able to go in sidebar yet? :)

Reply:
Beta tag isn’t in the Nav yet as it requires the change to be made by the Mosaic team. For now we’ll just rely on the tag on each page.